src/@orb.yml

```diff
@@ -1,11 +1,9 @@
-version: 2.1
-
 description: Google Kubernetes Engine (GKE) Orb
-
 display:
-  home_url: https://github.com/CircleCI-Public/gcp-gke-orb
```

Comment:
Should we leave home_url, but set to `https://cloud.google.com/kubernetes-engine`?

Reply:
Done!